docs/src/pages/discover-more/showcase/appList.js

```diff
@@ -40,7 +40,7 @@ const appList = [
     image: 'openclassrooms.jpg',
     link: 'https://openclassrooms.com/en/',
     similarWebVisits: 10190,
-    index: 34,
+    dateAdded: '2000-01-34',
```

Comment:
These dates are no valid ISO8601 dates, and cannot be parsed. (There is no 34. of January)
Might lead to errors in the future if someone ever tries to use the dates as dates instead of strings.

Reply:
> These dates are no valid ISO8601 dates, and cannot be parsed. (There is no 34. of January

They aren't intended to be – they're just intended to sort correctly. I could have manually adjusted the last few to start at the 1st of the following "month", but it didn't seem worth it, as they aren't valid dates to start with (i.e. they aren't the date the entry was added, so there's no further use case for them).

If it bothers you enough, you're welcome to fix them. :)